var/spack/repos/builtin/packages/hadoop/package.py

```diff
@@ -3,7 +3,7 @@
 # Produced at the Lawrence Livermore National Laboratory.
 #
 # This file is part of Spack.
-# Created by Alfredo Gimenez, gimenez1@llnl.gov, All rights reserved.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
```

Comment:
Updated a few packages that listed other authors. @tgamblin is this the right thing to do?